en/docs/reference/connectors/iso8583-connector/iso8583-inbound-endpoint-example.md

```diff
@@ -18,78 +18,101 @@ For example, while transferring bank and financial sector information using the
-       </parameters>
-   </inboundEndpoint>
-   ```
-   Sequence to process the message.
+2. Add a new **ISO8583 Inbound Endpoint** by clicking **+** icon in the `Inbound Endpoints` and select `ISO8583 WSO2 Listener`. 
+   
+    <img src="{{base_path}}/assets/img/integrate/connectors/iso8583-inbound.png" title="Creating inbound endpoint Listener selector" width="800" alt="Creating inbound endpoint"/>
+   
+3. Provide Inbound Endpoint Name and Port values as shown below and click `Create`.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/iso8583-inbound-form.png" title="Creating ISO8583 inbound endpoint form" width="800" alt="Creating ISO8583 inbound endpoint form"/>
+
+    The source view of the created inbound endpoint is shown below. 
+
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <inboundEndpoint name="ISOInboundEndpoint" class="org.wso2.carbon.inbound.iso8583.listening.ISO8583MessageConsumer" sequence="ISOInboundEndpoint-inboundSequence" onError="ISOInboundEndpoint-inboundErrorSequence" suspend="false">
+        <parameters xmlns="http://ws.apache.org/ns/synapse">
+            <parameter name="inbound.behavior">listening</parameter>
+            <parameter name="port">5001</parameter>
+            <parameter name="sequential">true</parameter>
+            <parameter name="coordination">true</parameter>
+            <parameter name="headerLength">0</parameter>
+            <parameter name="coreThreads">1</parameter>
+            <parameter name="maxThreads">3</parameter>
+            <parameter name="keepAlive">1</parameter>
+            <parameter name="queueLength">1</parameter>
+        </parameters>
+    </inboundEndpoint>
+    ```
    
-   In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
- 
-   ```xml
-   <?xml version="1.0" encoding="UTF-8"?><sequence xmlns="http://ws.apache.org/ns/synapse" name="requestISO" onError="fault">
-       <log level="full">
-           <property name="Log_Message for ISO8583 Inbound Endpoint" value="Message received from sample1-source"/>
+    !!! Note
+        When creating the ISO8583 inbound endpoint you have two options in defining the injecting sequence and error sequence.
+        <br/>- **Automatic**: Click the checkbox *Automatically generate sequence*
+        <br/>- **Manual**: You can select already defined sequences as injecting and error sequences.<br/>
+    
+    !!! Info
+        In this example, we are using `Automatic` option and that will create an injecting sequence named `ISOInboundEndpoint-inboundSequence` and error sequence named `ISOInboundEndpoint-inboundErrorSequence`.<br/>
+       
+4. Implement the injecting sequence which process the message - `ISOInboundEndpoint-inboundSequence`.
+   
+    In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
```

Comment:
```suggestion
    For simplicity, in this example you will just log the message. However, in a real-world use case, this could involve any type of message mediation.
```